### PR TITLE
refactor drop controller/model

### PR DESCRIPTION
**Pull request by berlintam**
@starkcoffee here's some refactoring! Let me know what you think.

### CODE REVIEW

File: app/views/drops/show.html.erb [hunk 1]
```diff
@@ -12,14 +12,17 @@
     <%= @drop.votes.count %>
   </div>
 </div>
+
 <div class ="drop-title">
-  <p><%= @drop_at_sc.title %></p>
+  <p><%= @drop.soundcloud_track.title %></p>
```
Comment (starkcoffee) [inline app/views/drops/show.html.erb]: I think also we store the title on the drop?

File: app/models/drop.rb [hunk 2]
```diff
@@ -1,10 +1,21 @@
...
 
   validates :sc_track, presence: true, numericality: { only_integer: true, message: "Sorry, this isn't a valid SoundCloud link. Try again!"}
   validates :place_id, presence: { message: "Please pick a place for your drop."}
+
+  def image_from(soundcloud_track)
+    if self.soundcloud_track.artwork_url.nil?
+      self.soundcloud_track.user.avatar_url.sub! 'large', 'crop'
+    else
+      self.soundcloud_track.artwork_url.sub! 'large', 'crop'
+    end
```
Comment (starkcoffee) [inline app/models/drop.rb:14]: Looks like the `soundcloud_track` argument is not being used, as we are using the soundcloud_track method? Also, for instance methods you can leave out the `self.` method. Or do you prefer to use `self.` for instance methods?

File: app/views/drops/show.html.erb (hunk 1 above)
Comment (berlintam) [inline app/views/drops/show.html.erb]: ah, you're right, I missed that.

File: app/models/drop.rb (hunk 2 above)
Comment (berlintam) [inline app/models/drop.rb:14]: ah yes, I see what you mean. strange how I did that... I'll adjust it.

**Pull request merged**